snmp/datadog_checks/snmp/data/default_profiles/_cisco-generic.yaml

```diff
@@ -86,6 +86,35 @@ metrics:
           OID: 1.3.6.1.4.1.9.9.13.1.5.1.2
           name: ciscoEnvMonSupplyStatusDescr
         tag: power_status_descr
+
+  - MIB: CISCO-ENVMON-MIB
+    table:
+      OID: 1.3.6.1.4.1.9.9.13.1.5
+      name: ciscoEnvMonSupplyStatusTable
+    symbols:
+      - name: ciscoEnvMonSupplyStatus
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.9.9.13.1.5.1.3
+          name: ciscoEnvMonSupplyState
+        tag: cisco_env_mon_supply_state
+        mapping:
+          1: normal
+          2: warning
+          3: critical
+          4: shutdown
+          5: notPresent
+          6: notFunctioning
+      - column:
+          OID: 1.3.6.1.4.1.9.9.13.1.5.1.4
+          name: ciscoEnvMonSupplySource
+        tag: power_source
+      - column:
+          OID: 1.3.6.1.4.1.9.9.13.1.5.1.2
+          name: ciscoEnvMonSupplyStatusDescr
+        tag: power_status_descr
+
```

Comment:
Left those tags just in case if we deprecate the previous table in the future

Reply:
I think those tags might be even necessary, otherwise, the `cisco_env_mon_supply_state` is not specific enough to distinguish each row of the table.